### PR TITLE
[WIP] Csv export (children list component)

### PR DESCRIPTION
**Pull request by jana47**
Issue: https://github.com/Aam-Digital/ndb-core/issues/126

### CODE REVIEW

File: src/app/children/children-list/children-list.component.ts [hunk 1]
```diff
@@ -88,6 +87,27 @@ export class ChildrenListComponent implements OnInit, AfterViewInit {
       .subscribe(results => this.prepareAttendanceData(results));
   }
 
+  getCsvExport() {
+    let csvData = '';
+    const rowNames = 'ProjectNumber,Name,Age,Date of Birth,Gender,Class,Center,Status,Mother Tongue,Religion,Address,Phone';
+    csvData += rowNames + '\r\n';
+    console.log(this.childrenList);
+    this.childrenList.map(child => {
+      csvData += child.projectNumber + ',' + child.name + ',' + child.age + ','
```
Comment (sleidig) [inline src/app/children/children-list/children-list.component.ts]: Did you have a look at the PapaParse library: https://www.papaparse.com/docs#json-to-csv ?

Using their functions we could probably avoid listing all the fields of the Child object explicitly (and having to update this code every time the Child class gets another attribute).

**Comment by sleidig** [comment]
@jana47 will you find the time to have another look at my remarks? Looking forward to merging this soon :-)

**Comment by Lorsbyjm** [comment]
@sleidig Jana is not doing any work for the team right now. So feel free to merge, if it is ok.

**Comment by sleidig** [comment]
should use PapaParse as that can simplify the code a little and as a well maintained library is probably more reliable for all kinds of edge cases

**Comment by TheSlimvReal** [comment]
closed in favor of the new PR #352

**Pull request closed without merging**